cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamFetchHandler.java

```diff
@@ -109,7 +110,8 @@ public void fetch(HttpRequest request, HttpResponder responder,
       return;
     }
 
-    StreamConfig streamConfig = streamAdmin.getConfig(stream);
+    Id.Stream streamId = Id.Stream.from(accountID, stream);
```

Comment:
I will get the namespaceId from the request once v3 handlers are implemented (separate PR).